src/components/cell-control/input.md

```diff
@@ -0,0 +1,45 @@
+A grid cell for text datum. 
+
+```jsx
+import Input from '@mavenlink/design-system/src/components/cell-control/input.jsx';
+
+const ids = {
+  th: uuid.v4(),
+};
+
+<table role="grid">
+  <thead>
+    <tr>
+      <th>State</th>
+      <th id={ids.th} role="columnheader" scope="col">Example</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td role="rowheader">Default</td>
+      <Input labelledBy={ids.th} id={uuid.v4()} />
+      <Input labelledBy={ids.th} id={uuid.v4()} />
```

Comment:
The third-column is just dummy cells to see how it behaves with at least a 3x3 grid. I think with more cell controls, we can make a more interesting example.